docker/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM php:7.4-apache
+
+ARG BRANCH=stable
+
+COPY ./entrypoint.sh /usr/local/sbin/entrypoint
```

```suggestion
COPY --chmod=755 ./entrypoint.sh /usr/local/sbin/entrypoint
```

no need to have two layers (COPY and RUN) with recent enough docker.